apps/evm/lib/evm/machine_state.ex

```diff
@@ -30,7 +30,7 @@ defmodule EVM.MachineState do
           memory: memory,
           active_words: integer(),
           stack: Stack.t(),
-          last_return_data: [EVM.val()]
+          last_return_data: [EVM.val()] | nil | EVM.val()
```

Comment:
Is there a way to normalize this data in this module to always return a list (could be empty), so that you don't have to normalize it in `read_return_data` in the other file?

Reply:
done